contrib/errorprone/src/python/pants/contrib/errorprone/register.py

```diff
@@ -0,0 +1,16 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+from pants.goal.goal import Goal
+from pants.goal.task_registrar import TaskRegistrar as task
+
+from pants.contrib.errorprone.tasks.errorprone import ErrorProne
+
+
+def register_goals():
+  Goal.register('errorprone', 'Check Java code for Error Prone violations.')
```

Comment:
Yeah, you're right.  The same unused line was in the findbugs/register.py so I removed it there as well.